layers/meta-tegrademo/scripts/setup-udev-rules

```diff
@@ -25,8 +25,18 @@ SUBSYSTEMS=="usb", ATTRS{idVendor}=="0955", ATTRS{idProduct}=="7019", GROUP="plu
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="0955", ATTRS{idProduct}=="7e19", GROUP="plugdev"
 # Jetson Nano
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="0955", ATTRS{idProduct}=="7f21", GROUP="plugdev"
-# Jetson Orin AGX
+# Jetson Orin AGX (P3701-0000 Developer Kit module), P3701-0005 with 64GB), (P3701-0008 with 64GB)
```

Comment:
**Minor:** Please apply a minor update 
`# Jetson Orin AGX (P3701-0000 Developer Kit module), (P3701-0005 with 64GB), (P3701-0008 with 64GB)`

Reply:
Fixed the missing."("